ruby_on_rails/cucumber.md

```diff
@@ -0,0 +1,86 @@
+# Cucumber
+
+[Cucumber](https://cucumber.io) is a testing framework which allows us to specify feature tests using plain language.
+In our projects, we use Cucumber for e2e tests. We integrate it using the
```

Comment:
I wouldn't state that since is not true. Also this is still the optional gems people can use. I would remove that sentence: "In our projects, we use Cucumber for e2e tests.". We also use rspec in some projects.